### PR TITLE
Review load is a full science + vehicle load but only vehicle commands executed.

### PR DESCRIPTION
**Pull request by Gregg140**
Handles the circumstance where the Review load is a full science + vehicle
commands load, but we know - prior to the review - that only the vehicle load command set, in the
review load, will be executed. The new LR switch: VOR for Vehicle Only Review was implemented.

### CODE REVIEW

File: lr [hunk 1]
```diff
@@ -652,19 +660,56 @@ open(CONTFILE, ">$ofls_dir/ACIS-Continuity.txt");
...
     # User selected VOC for the VO switch. Tell the model to read
     # the vehicle only load for the Continuity. Append "VO_"
     # to the recognized load type
     $load_type = "VO_".$load_type;
   }
 
+# Now if $VO_choice is either VOB or VOR, then the REVIEW Load is to be run Vehicle Only.
+# In this case copy the CR*.backstop file to ORIG_CR*.backstop, and the .../vehicle/VR*.backstop
+# is to be copied to CR*_V.backstop.
+# We do this by calling the Python program Swap_Baclkstop_Files.py
```
[the remark this answers is not in the document]
Comment (jazan12) [reply lr:685]: ("baclkstop")

File: CHECK_POWER_COMMANDS/Check_Power_Cmds.py [hunk 2]
```diff
@@ -201,32 +205,40 @@ def run_one_command(cmd, system_state, bfc, rule_set):
...
-# first ACISPKT.  Then you process that first one.  This will set up
-# the system state for processing the next ACISPKT that you encounter
-while present_cmd['cmd_type'] != 'ACISPKT':
 
-    # Run the ORB rules on the present state
-    system_state, new_rules_fired, violations_list = run_one_command(present_cmd,
-                                                                     system_state,
-                                                                     bfc,
-                                                                     Rulesets.ORB_CMD_SW_rule_set)
+# If this s a Vehicle-Only Review load then there will be no ACISPKT commands within the load.
```
Comment (jazan12) [inline CHECK_POWER_COMMANDS/Check_Power_Cmds.py:209]: typo: "is"

File: CHECK_POWER_COMMANDS/Check_Power_Cmds.py [hunk 3]
```diff
@@ -201,32 +205,40 @@ def run_one_command(cmd, system_state, bfc, rule_set):
...
+# So skip the while loop which is intended ot take you to the first one.  
+# This means that array_row_number will equal 0
+
+if 'ACISPKT' in system_packets['cmd_type']:
```
Comment (jazan12) [inline CHECK_POWER_COMMANDS/Check_Power_Cmds.py:213]: I expected to see a violation raised any time an acispkt is found in a VO backstop but don't see this is done.  Can you confirm?

**Review by jazan12 — COMMENTED** [review]
Approved if you confirm on the one question raised.

File: CHECK_POWER_COMMANDS/Check_Power_Cmds.py (hunk 3 above)
Comment (Gregg140) [reply CHECK_POWER_COMMANDS/Check_Power_Cmds.py:213]: The purpose is not to check if the VO file has ACISPKT commands. The purpose is to not crash searching for ACISPKT commands when there aren't any. Remember that in an assembled history, there can be ECS measurements and/or power commands set via CAP so there MAY actually be ACISPKT commands within the VO time period but which did not appear in the VO*.backstop file.

**Comment by jazan12** [comment]
I still would think we want a check that there are no ACIS commands in a VO
load, or at least that these are flagged explicitly as warnings (in the way
that other LR warnings print potential issues); particularly since in the
cases I've seen for VO loads I wouldn't think we should see any ACIS
commanding taking place.  (Things like LTS ECS that we run by CAP should
only appear afterwards in history.)

-Jack

**Review by jazan12 — APPROVED** [review]

**Review by jzuhone — APPROVED** [review]

**Pull request merged**